aioredisqueue/queue.py

```diff
@@ -10,9 +11,11 @@ def __init__(self, redis, key_prefix='aioredisqueue_', *,
                  fetching_fifo_key=None,
                  payloads_hash_key=None,
                  ack_hash_key=None,
+                 last_requeue_key=None,
                  task_class=task.Task,
                  lua_sha=None,
-                 loop=None):
+                 loop=None,
+                 requeue_interval=10000):
```

Comment:
there should be a way to opt it out. Perhaps 0 value should indicate that we don't do requeueing in this instance.